style.css

```diff
@@ -4,6 +4,7 @@
   min-height: 100vh;
   flex-direction: column;
   background-color: white;
+  background-image: url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');
```

Comment:
- [ ] Please consider implementing these and the previous reviewers requested changes.
- The content should be aligned with flexbox.
- This gives us a lot of options to align all our elements, plus, it's responsive.
- The section should have the margins (top, bottom, left, and right) defined in the figma design.
- The section should use the correct font and colors (background and font) defined for the [template]
you design
![qwe](https://user-images.githubusercontent.com/96683843/212977866-2e21d2e1-0b6c-4442-9765-9ae20eb11627.jpg)

figma design
![qwer](https://user-images.githubusercontent.com/96683843/212977875-1fd737a3-bd51-4ef0-8005-5a3766942776.jpg)

Reply:
I have made all this changes, I don't know why it's not being visible on your side